### ISSUE TITLE
Some Stocks endpoints return incorrectly

### ISSUE THREAD

**Issue by addisonlynch** [addisonlynch/iexfinance#34]
It appears that some endpoints (such as ``get_earnings`` are returning incorrectly-formatted data)

```python

from iexfinance import Stock

aapl = Stock("AAPL", output_format='pandas')
aapl.get_earnings()

#                                                                            AAPL
# earnings [{'fiscalPeriod': 'Q4 2017', 'announceTime': '...
# symbol                                                                     AAPL
```
This is incorrect. In addition, odd behavior is that ``get_earnings`` sometimes returns with a symbol index, and sometimes does not. This would cause calls like ``aapl.get_earnings()["totalDebt"]`` to fail.

**Comment by addisonlynch** [comment]
The Stocks Earnings and Financials endpoints return differently than the rest:

```

**Comment by addisonlynch** [comment]
The Stocks/earnings and Stocks/financials endpoints return differently than the rest of the group, in that they have an extra dictionary level:

```
{
  "symbol": "AAPL",
  "earnings": [
    {
      "actualEPS": 2.1,
      "consensusEPS": 2.02
      ...
```

This causes some requests to return a symbol-indexed dictionary of the endpoint instead of the actual endpoint. This also causes pandas format to be incorrect.

**Comment by addisonlynch** [comment]
Merged #35.
